Luke7787/ImmaculateInventors
# Feature/UI spec

Summary:
- developed the header spec
- created a clickable sign in button
- sign in modal has fields that can be filled out

Link to any specs used:
[](https://www.figma.com/file/AE3t3fUj0nMRRtUQX3BBbT/UI-Design---Sign-In-Modal?type=design&node-id=0-1&mode=design&t=5i95hwfEXQilawgl-0)

[](https://www.figma.com/file/XP0UJUNIP8dEv6lqgWMQEg/UI-Design?type=design&mode=design&t=5i95hwfEXQilawgl-0)

How to test:
npm test
verify the following
- design matches spec
- sign in modal can be opened and closed
- can enter text within the input boxes
- text clears when modal closes
- text clears upon submission

Notes:
backend not connected yet
